src/Hoard/Adapter/Memcached.php

```diff
@@ -30,8 +30,34 @@ class Memcached extends \Hoard\AbstractAdapter
             if(!class_exists('\Memcached')) {
                 throw new \Exception('Memcached extension is not installed.');
             }
+
             $this->connection = new \Memcached();
 
+            // setup memcached options
```

Comment:
Please add documentation to the Readme.md so that we know what options this can apply.